api/rest/v1/bulk_observations_point_linked.md

```diff
@@ -0,0 +1,298 @@
+---
+layout: default
+title: Single Observation (linked)
+nav_order: 9
+parent: REST (v1)
+grand_parent: API
+published: false
+permalink: /api/rest/v1/bulk/observations/point/linked
+---
+
+# /v1/bulk/observations/point/linked
+
+Retrieve single [observations](/glossary.html#observation) of multiple [variables](/glossary.html#variable) at a set date for entities linked to a parent entity by the same property.
+
+This is useful for retrieving observations for all places within a parent place. For example, this could be getting the population of women in 2018 for all states in the United States.
+
+<div markdown="span" class="alert alert-info" role="alert" style="color:black; font-size: 0.8em">
+   <span class="material-icons md-16">info </span><b>Note:</b><br />
+   Currently, this endpoint only supports the `containedInPlace` property and `Place` entities. Support for other properties and entity types will be added in the future.
+</div>
+
+<div markdown="span" class="alert alert-warning" role="alert" style="color:black; font-size: 0.8em">
+    <span class="material-icons md-16">info </span><b>See Also:</b><br />
+    To get single observations for a single place, see [/v1/observations/point](/api/rest/v1/observations/point).<br />
+    To get a series of observations for all places within a parent place, see [/v1/bulk/observations/series/linked](/api/rest/v1/observations/series/linked).
+</div>
+
+## Request
+
+<div class="api-tab">
+  <button id="get-button" class="api-tablink" onclick="openTab(event, 'GET-request')">GET Request</button>
+  <button id="post-button" class="api-tablink" onclick="openTab(event, 'POST-request')">POST Request</button>
+</div>
+
+<div id="GET-request" class="api-tabcontent api-signature">
+https://api.datacommons.org/v1/bulk/observations/point/linked?linked_property=containedInPlace&linked_entity={parent_place_dcid}&entity_type={place_type}&variables={variable_dcid_1}&variables={variable_dcid_2}&date={date}&key={your_api_key}
+</div>
+
+<div id="POST-request" class="api-tabcontent api-signature">
+URL:
+https://api.datacommons.org/v1/bulk/observations/point/linked
+
+Header:
+X-API-Key: {your_api_key}
+
+JSON Data:
+{
```

Comment:
maybe add ```json fence here?

Reply:
the ```json and ``` will show up as raw text if I do so (that div behaves like a <pre> or <code> tag)